mpv/scripts/run_websocket_server_windows.lua

```diff
@@ -6,17 +6,39 @@ local utils = require 'mp.utils'
 local config_file_path = mp.find_config_file("mpv.conf")
 local config_folder_path, config_file = utils.split_path(config_file_path)
 local mpv_websocket_path = utils.join_path(config_folder_path, "mpv_websocket.exe")
+local initialised_websocket
 
-mp.command_native_async({
-    name = "subprocess",
-    playback_only = false,
-    capture_stdout = true,
-    capture_stderr = true,
-    args = {
-        mpv_websocket_path,
-        "-m",
-        "\\\\.\\pipe\\tmp\\mpv-socket",
-        "-w",
-        "6677",
-    },
-})
+local function start_websocket()
+	initialised_websocket = mp.command_native_async({
+		name = "subprocess",
+		playback_only = false,
+		capture_stdout = true,
+		capture_stderr = true,
+		args = {
+			mpv_websocket_path,
+			"-m",
+			"\\\\.\\pipe\\tmp\\mpv-socket",
+			"-w",
+			"6677",
+		},
+	})
+end
+
+local function end_websocket()
+	mp.abort_async_command(initialised_websocket)
+	initialised_websocket = nil
+end
+
+local function toggle_websocket()
+	local paused = mp.get_property_bool('pause')
+	if initialised_websocket and paused then
+		mp.osd_message('end websocket')
```

Comment:
@peldas I assume this displays this message on mpv. Do we want to propagate this to the linux/mac script as well? Or should we delete this?